Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalMinBias_Output_cff.py

```diff
@@ -9,7 +9,10 @@
         SelectEvents = cms.vstring('pathALCARECOHcalCalMinBias')
     ),
     outputCommands = cms.untracked.vstring(
-	'keep *_gtDigisAlCaMB_*_*',
```

Comment:
ok, may you please re-do the measurement of event/size on average
after you've updated the event content ?
Thanks!